bika/lims/utils/analysisrequest.py

```diff
@@ -195,6 +186,46 @@ def get_sample_from_values(context, values):
     return sample
 
 
+def get_services_uids(context=None, analyses_serv=[], values={}):
```

Comment:
@Espurna using mutable values like [] or {} for the default values of function parameters, will make terrible awful errors happen in your future.  I have changed it, but I thought you should know for your sanity..